web/models.py

```
@@ -2804,3 +2805,417 @@ class ScheduledPost(models.Model):
+    chapter = models.ForeignKey(Chapter, on_delete=models.CASCADE, related_name='members')
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='chapter_memberships')
+    role = models.CharField(max_length=20, choices=ROLE_CHOICES, default='participant')
+    status = models.CharField(max_length=20, choices=STATUS_CHOICES, default='pending')
+    joined_at = models.DateTimeField(auto_now_add=True)
+    approved_at = models.DateTimeField(null=True, blank=True)
+    approved_by = models.ForeignKey(
+        User,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='approved_memberships'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)